api/v1beta2/kyma_types.go

```diff
@@ -387,24 +387,42 @@ func (kyma *Kyma) IsBeta() bool {
 
 type AvailableModule struct {
 	Module
-	Enabled bool
-	Valid   bool
+	Enabled         bool
+	ValidationError string
```

Comment:
why not declare it as `error`?

Reply:
Unfortunately this makes controller-tools generator complain: `src/github.com/kyma-project/lifecycle-manager/api/v1beta2/kyma_types.go:392:2: invalid field type: interface{Error() string}`
I haven't found any easy solution yet.

Reply:
did you also fixed related code regarding `ValidationError`? you have many place assign it with string.

Reply:
I'll try to move the function and the type out of the api package.

Reply:
moved